manager/state/raft/storage.go

```diff
@@ -25,10 +25,14 @@ import (
 
 var errNoWAL = errors.New("no WAL present")
 
-func (n *Node) walDir() string {
+func (n *Node) legacyWALDir() string {
```

Comment:
Most code only has to deal with the current version, so I don't want to leak the versioning stuff to callers of `walDir`. But I could rename `legacyWALDir` to `v2WALDir` if you think that's better (and so on with `snapshotDir`).